01_draw_functions/01_subfunctions/SensorPlots.m

```diff
@@ -9,9 +9,9 @@ function SensorPlots(sysvector, topics, fconv_gpsalt)
     fig4.Name = 'Sensor Data';
     raw(1)=subplot(4,1,1);
     hold on;
-    plot(sysvector('sensor_combined_0.magnetometer_ga_0').Time,sysvector('sensor_combined_0.magnetometer_ga_0').Data);
-    plot(sysvector('sensor_combined_0.magnetometer_ga_1').Time,sysvector('sensor_combined_0.magnetometer_ga_1').Data);
-    plot(sysvector('sensor_combined_0.magnetometer_ga_2').Time,sysvector('sensor_combined_0.magnetometer_ga_2').Data);
+    plot(sysvector('sensor_mag_0.x').Time,sysvector('sensor_mag_0.x').Data);
```

Comment:
Can you use here 'vehicle_magnetometer', the 'sensor_mag_0' is used in the RawSensorPlots.

Reply:
This was also not logged during the FAT flight, which is why I used sensor_mag_0. Feel free to change this, but then we also need to add that to the logging.txt.